src/test/java/teammates/test/pageobjects/InstructorHomePage.java

```diff
@@ -164,6 +164,30 @@ public InstructorHomePage clickFeedbackSessionPublishLink(String courseId, Strin
         return changePageType(InstructorHomePage.class);
     }
 
+    public void clickResendPublshedLinksLink(String courseId, String evalName) {
+        click(getResendPublishedLinksLink(courseId, evalName));
+        ThreadHelper.waitFor(1000);
+    }
+
+    public void cancelResendPublishedLinksForm() {
```

Comment:
I realised most of the new methods you added are almost identical to the remind particular students ones. Any hope of modifying the existing ones to be more general so they can be reused?

Reply:
Yes, most of them are almost identical, except for the `id` of the element being processed. If they are generalised though the tests will have to pass the element `id` every time the functions are called, personally I think it's quite easy to make mistake, unless the elements' `ids` are put down as constants somewhere.

Reply:
Can write a general method in `AppPage`, then use it for these specific methods in `InstructorHomePage`. Also, the `id` can be included in a `@FindBy` annotation at the top of `InstructorHomePage` - take a look at the other elements there. Can do the same for both modals.